app/common/state/windowState.js

```diff
@@ -148,6 +148,13 @@ const api = {
     return state.set('windows', windows.delete(index).insert(index, windowValue))
   },
 
+  updateComputedStyles: (state, action) => {
```

Comment:
I think `setComputedStyles` and we are moving away from passing actions into state helpers so I think this should take `windowId` and `computedStyles`